packages/cli/src/commands/deploy/index.ts

```diff
@@ -522,7 +522,12 @@ export default async (client: Client): Promise<number> => {
       quiet,
       wantsPublic: argv['--public'] || localConfig.public,
       type: null,
-      nowConfig: localConfig,
+      nowConfig: {
+        ...localConfig,
+        // `images` is allowed in "vercel.json" and processed
+        // by `vc build`, but don't send it to the API endpoint
+        images: undefined,
```

Comment:
If we dont send it to the API, how do we make it work with the deployment?

Are you thinking `.vercel/output/config.json` is going to be read by the API instead?

Reply:
It's still in the `vercel.json` source file, which is read during build by `vc build`.